Source/AsyncGenerator.Tests/AsyncGenerator.Tests.csproj

```diff
@@ -25,6 +25,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.6.0" />
     <PackageReference Include="NUnit" Version="3.10.1" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.10.0" />
+    <PackageReference Include="SQLitePCLRaw.core" Version="1.0.0" /> <!--Needed for MONO 5.16-->
```

Comment:
👍

Reply:
I didn't saw that you had the same problem in nhibernate/nhibernate-core/pull/1830, I need to add this reference also for the commandline project.

Reply:
The reference is now added also for the commandline project so the workaround should not be needed anymore.

Reply:
Thanks a lot.